draft-tigress-requirements.md

```diff
@@ -73,6 +73,79 @@ TODO Introduction
 
 {::boilerplate bcp14-tagged}
 
+General terms:
+
+- Credential information - data used to authenticate the user with an access point.
+- Provisioning information* - data transferred from Sender to Receiver device that is both necessary and sufficient for the Receiver to request a new credential from Provisioning Partner to provision it to the Receiver device.
+- Provisioning - A process of adding a new credential to the device.
+- Provisioning Partner - an entity which facilitates Credential Information lifecycle on a device. Lifecycle may include provisioning of credential, credential termination, credential update.
+- Sender (device) - a device initiating a transfer of Provisioning Information to a Receiver that can provision this credential.
+- Receiver (device) - a device that receives Provisioning Information and uses it to provision a new credential.
+- Intermediary (server) - an intermediary server that facilitates transfer of provisioning information between Sender and Receiver.
+
+# Use Cases
+
+## 1. Let's say Ben owns a vehicle that supports digital keys which comply with the CCC open standard. Ben would like to let Ryan borrow the car for the weekend. Ryan and Ben are using two different mobile phones with different operating systems. In order for Ben to share his car key to Ryan for a weekend, he must transfer some data to the receiver device. The data structure shared between the two participants is defined in the CCC. In addition, the CCC requires the receiver to generate required key material and return it to the sender to sign and return back to the receiver. At this point, the receiver now has a token that will allow them to provision their new key with the car.
```

Comment:
You don't want to put a `##` here. The formatting looks very weird, see https://dimmyvi.github.io/tigress-requirements/Add-use-cases-and-terminology/draft-tigress-requirements.html#name-1-lets-say-ben-owns-a-vehic

Reply:
Thanks for catching this, I should've looked at the preview generating. Removing numbers

Reply:
@dimmyvi no, the problem isn't the numbers, it's the `##`. That marks these lines as headers.

Reply:
fixed